lib/auth/auth_test.go

```diff
@@ -590,20 +590,12 @@ func requireTokenExpiry(t *testing.T, token types.ProvisionToken, expectExpiry t
 }
 
 func TestTokensCRUD(t *testing.T) {
-	// TODO(noah): completely refactor this test suite when deprecating
-	// GenerateToken - break this down into separate tests per method rather
-	// than grouping all CRUD methods.
+	// This test is deprecated, prefer creating tests named after the RPC
+	// in tls_test.go . See TestCreateToken for an example.
```

Comment:
nit: I don't think this comment helps - I'd either take a TODO or accept the test as-is.

+1 for not having "CRUD" tests, though.

Reply:
fb8f573e25